docs/ispc.rst

```diff
@@ -1524,7 +1524,7 @@ The following reserved words from C89 are also reserved in ``ispc``:
 ``ispc`` additionally reserves the following words:
 
 ``bool``, ``delete``, ``export``, ``cdo``, ``cfor``, ``cif``, ``cwhile``,
-``false``, ``foreach``, ``foreach_active``, ``foreach_tiled``,
+``false``, ``float16``, ``foreach``, ``foreach_active``, ``foreach_tiled``,
 ``foreach_unique``, ``in``, ``inline``, ``noinline``, ``__vectorcall``, ``int8``, ``int16``,
```

Comment:
Looks like we missed `uint*` identifiers here, would be good to add them as well.

Reply:
Added.